aos_env/Dockerfile

```diff
@@ -1,20 +1,21 @@
 # Dockerfile to execute a base machine with all set to test active optics
 # system software
 
-FROM lsstsqre/centos:7-stack-lsst_distrib-w_2020_15
+FROM lsstsqre/centos:7-stack-lsst_distrib-w_latest
 
 LABEL author="Te-Wei Tsai <ttsai@lsst.org>"
 
+ARG SIMS_VERSION
+
 ARG LSST_USER=lsst
 ARG LSST_USER_HOME=/home/$LSST_USER
 ARG LSST_DIR=/opt/lsst/software/stack
-ARG SIMS_VERSION=sims_w_2020_15
 ARG SHEBANGTRON_URL=https://raw.githubusercontent.com/lsst/shebangtron/master/shebangtron
 
 USER root
 
 # Install the required packages
-RUN yum install -y csh graphviz\
+RUN yum install -y csh graphviz wget\
```

Comment:
Centos comes with curl, which is preferred (wget problem is apparently it lacks C library, and is considered depreciated). curl -O does the same job as wget (you need to provide -O). I strongly recommend to remove wget and use curl instead.

Reply:
Done in commit: b5ca019. Thanks!